apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  IconButton,
+  Text,
+  Separator,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { getFormattedAspectRatio } from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  Cross1Icon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+import prettyBytes from "pretty-bytes";
+
+const truncatedText = {
+  position: "relative",
+  maxWidth: 80,
+
+  div: {
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+
+    "&:after": {
```

Comment:
single colon is for pseudos
```suggestion
    "&::after": {
```